gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.12-bin.zip
```

Comment:
instead of these changes, should check this parent repo, and merge update for change logs
[Chucker](https://github.com/ChuckerTeam/chucker/blob/main/CHANGELOG.md)